index.js

```diff
@@ -1035,6 +1143,41 @@
     };
   }());
 
+  //# lte :: (a, b) -> Boolean
+  //.
+  //. Returns `true` if its arguments are of the same type and less than or
+  //. equal according to the type's [`fantasy-land/lte`][] method; `false`
+  //. otherwise.
+  //.
+  //. `fantasy-land/lte` implementations are provided for the following
+  //. built-in types: Null, Undefined, Boolean, Number, Date, String,
+  //. Array, Arguments, Error, and Object.
+  //.
+  //. The algorithm supports circular data structures. Two arrays are lte
+  //. if they have the same index paths and for each path have lte values.
+  //. An array which represents `[1, [1, [1, [1, [1, ...]]]]]` and one that
+  //. represents `[2, [2, [2, [2, [2, ...]]]]]`, for example,
+  //. are lte even if their internal structures differ. Two objects are lte
+  //. if they have the same property paths and for each path have lte values.
```

Comment:
Let's shorten this paragraph to something like this:

_The algorithm supports circular data structures in the same manner as [`equals`][1]._


[1]: https://github.com/sanctuary-js/sanctuary-type-classes#equals